src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -266,37 +268,38 @@ export function getEventContent(event) {
       };
     case EVENT_TYPES.bvaRemand:
```

Comment:
Not really 100% related to this PR but I noticed that the value for the `bvaRemand` is misspelled in the EVENT_TYPES map.

Reply:
Oh, good catch!

Reply:
Fixed https://github.com/department-of-veterans-affairs/vets-website/pull/7155